ios/LayoutReanimation/REAAnimationsManager.m

```diff
@@ -279,20 +279,20 @@ - (void)maybeDropAncestors:(UIView *)child
   }
 }
 
-- (BOOL)removeRecursive:(UIView *)view fromContainer:(UIView *)container withoutAnimation:(BOOL)removeImmediately;
+- (BOOL)startAnimationsRecursive:(UIView *)view removingSubviewsWithoutAnimations:(BOOL)removeSubviewsWithoutAnimations;
```

Comment:
There's an inconsistency in the method signature - `removing` vs. `remove`. I'd like to suggest renaming both to `shouldRemoveSubviewsWithoutAnimations` ("remove" is a verb and those are commonly used for method names, while "should" indicates that it's a flag).